.github/workflows/auto-assign.yml

```diff
@@ -0,0 +1,15 @@
+name: "Auto Assign PR"
```

Comment:
I thought the job should run automatically. Is there anything I need to do to test that it works?